### DISCUSSION TITLE
不要局限于翻译

### DISCUSSION THREAD

**Discussion by Alpha211201** [nextai-translator/nextai-translator#449] [Ideas]
目前这个全局可以快捷键呼出的方式挺方便的，所以能不能把gpt对话的功能也打开，这比要开个网页快很多。

**Comment by AnYiEE** [comment]
可这是translator。

**Comment by szh-bash** [comment]
感觉还是 focus 翻译有助于插件发展，全功能插件那又是一个新项目了XD

**Comment by evolution9502** [comment]
人家都把潤色、總結、分析做出來了，真要問問題直接上網頁版不好嗎
重點是插件需要API key，也就是意謂著一旦用量到一定程度之後是要付費的
你就不怕問著問著下個月帳單突然多個幾百幾千?
